index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="style.css">
+
+  <title>Project-X</title>
+</head>
+<section>
+  <div class="notif-bar">
+    <h4>God-Logic</h4>
+    <img src="images/hamburger.png" alt="hamburger" class="hamburger">
+  </div>
+  
+</section>
+
+<section>
+  <div class="introduction">
+    <div class="intro-Dskt">
+      <h1>Hey There, I am Maboela</h1>
+      <h3>I’am a Software Developer</h3>
+      <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+  don’t hestiate to contact me.</p>
+    </div>
+    
+    <img src="images/social line.png" alt="social line" class="social">
```

Comment:
- [ ] Kindly note that these social icons should be added individually and not as one image. You can download the images individually. When you download them, kindly use the <ul> tag and use a <li> tag individually in them. in that li tag, don't forget to use a <a> tag because these are going to be linked to your social media accounts, so the href in the <a> tag would help with that. Then you are good to go on it.